contracts/transfer/src/state.rs

```diff
@@ -391,6 +417,14 @@ impl TransferState {
         self.tree.root()
     }
 
+    /// Get the opening
+    pub fn opening(
+        &self,
+        pos: u64,
+    ) -> Result<Option<PoseidonBranch<TRANSFER_TREE_DEPTH>>, Error> {
+        Ok(self.tree.opening(pos)?)
+    }
+
```

Comment:
I don't understand why this is introduced since it is seemingly unused.

Reply:
I is used in TransferWrapper prepare_execute, I'll leave it for now